README.md

```diff
@@ -39,6 +39,89 @@ bundle install
 rake -T
 ```
 
+## Dockerfile
+
+### Build image
+```
+docker build -t <image_name> .
```

Comment:
replace here and in other places replace <image_name> to something real, just copy/paste and use.